### PR TITLE
Gold req

### PR DESCRIPTION
**Pull request by caoash**
Add two gold problem suggestions.

### CODE REVIEW

**Review by dongliu0426 — APPROVED** [review]

**Review by dolphingarlic — CHANGES_REQUESTED** [review]
Use tabs instead of spaces

File: content/4_Gold/Intro_DP.mdx [hunk 1]
```diff
@@ -47,6 +47,15 @@ export const problems = {
 			['DP'],
 			'dp[i][j] -> minimum sum of net sizes needed to catch m snakes with k changes, precompute maximums'
 		),
+		new Problem(
+			'Gold',
+			'Angry Cows',
+			'597',
+			'Easy',
+			false,
+			['DP', '2P', 'Binary Search'],
```
Comment (jessechoe10) [inline content/4_Gold/Intro_DP.mdx:56]: Should we keep or remove from Binary Search module?

Comment (frodakcin) [reply content/4_Gold/Intro_DP.mdx:56]: Good catch.... I don't know the problem -- if it involves dp why is it in the binsearch module, and why is it starred there?

Comment (caoash) [reply content/4_Gold/Intro_DP.mdx:56]: It's bsearch on ans + DP to check.

**Pull request merged**